components/net/image_cache.rs

```diff
@@ -55,14 +55,15 @@ fn set_webrender_image_key(compositor_api: &CrossProcessCompositorApi, image: &m
         return;
     }
     let mut bytes = Vec::new();
+    let frame_byte = image.get_bytes();
```

Comment:
I think `frame_bytes` is a better name as there is usually more than a single byte!